### PR TITLE
List Demos small updates

### PR DESCRIPTION
**Pull request by flagmanAndrew**
all list updates

### CODE REVIEW

File: JSDemos/Demos/List/ItemDragging/jQuery/index.js [hunk 1]
```diff
@@ -2,22 +2,23 @@ $(function() {
 
     function createList(selector, tasks) {
         var list = $(selector).dxList({
```
Comment (ksercs) [inline JSDemos/Demos/List/ItemDragging/jQuery/index.js]: seems like we can get rid of `list` variable at all

File: JSDemos/Demos/List/ListEditingAndAPI/React/App.js [hunk 2]
```diff
@@ -10,63 +10,46 @@ class App extends React.Component {
   constructor() {
     super();
     this.state = {
-      allowDeleting: false,
+      allowDeletion: false,
       deleteType: 'toggle',
```
Comment (ksercs) [inline JSDemos/Demos/List/ListEditingAndAPI/React/App.js]: shouldn't we replace "deleting type" with "itemDeleteMode" everywhere?

**Comment by ksercs** [comment]
I did not check that necessary changes were made in all appropriate files. I gonna trust you about this :)

File: JSDemos/Demos/List/ListEditingAndAPI/React/App.js (hunk 2 above)
Comment (flagmanAndrew) [reply JSDemos/Demos/List/ListEditingAndAPI/React/App.js]: Yes. But somebody already did it for me.
All labels above SelectBoxes are "Item delete mode". I though this task is about labels (what customers see on a page)

**Review by ksercs — DISMISSED** [review]

File: JSDemos/Demos/List/ListEditingAndAPI/React/App.js (hunk 2 above)
Comment (ksercs) [reply JSDemos/Demos/List/ListEditingAndAPI/React/App.js]: Customers see the code too)

**Review by ksercs — APPROVED** [review]

**Pull request merged**